guides/request-validation-django/example-3/example-3.5.x.py

```diff
@@ -8,22 +8,24 @@
 
 def validate_twilio_request(f):
     """Validates that incoming requests genuinely originated from Twilio"""
+
     @wraps(f)
     def decorated_function(request, *args, **kwargs):
         # Create an instance of the RequestValidator class
         validator = RequestValidator(os.environ.get('TWILIO_AUTH_TOKEN'))
 
         # Validate the request using its URL, POST data,
         # and X-TWILIO-SIGNATURE header
-        request_valid = validator.validate(
-            request.build_absolute_uri(),
-            request.POST,
-            request.META.get('HTTP_X_TWILIO_SIGNATURE', ''))
+        request_valid = validator.validate(request.build_absolute_uri(),
+                                           request.POST,
+                                           request.META.get(
```

Comment:
Indeed! I also tried to fix some snippets that were like _bad_ formatted, but I think it's better to have a different PR to handle those things.